core-text/src/font.rs

```diff
@@ -983,7 +984,9 @@ fn out_of_range_variations() {
     // attributes greater than max are dropped on macOS <= 11
     // on macOS 12 they seem to be preserved as is.
     let var_attrs = var_attrs.find(variation_attribute);
-    if macos_version() >= (12, 0, 0) && macos_version() < (13, 0, 0) {
+
+    // Just guessing -- this passed on (14, 4, 1)
+    if version >= (14, 0, 0) {
```

Comment:
Changing this first case makes the comment above this not make any sense.

Reply:
Fair -- I updated the comment to be more general.